handler_creator.go

```diff
@@ -9,7 +9,13 @@ func handlerByMethod(route *string, do interface{}, method string) *handler {
 	handlerDo := reflect.ValueOf(do)
 	h := newHandler(handlerDo)
 	h.method = method
-	h.routes = strings.Split(strings.Trim(*route, "/"), "/")
+
+	h.routes = make([]string, 0)
+	for _, r := range strings.Split(strings.Trim(*route, "/"), "/") {
```

Comment:
I think we should create a helper function to generate the route array for us.
because this it is the mistake will easy to make.